package.json

```diff
@@ -1,93 +1,115 @@
 {
-  "name": "@storyblok/svelte-ws",
-  "private": true,
-  "version": "0.0.1",
-  "workspaces": [
-    "./lib",
-    "./playground",
-    "./playground-sveltekit"
+  "name": "@storyblok/svelte",
+  "type": "module",
+  "version": "5.0.0",
+  "private": false,
+  "packageManager": "pnpm@9.15.3",
+  "description": "SDK to integrate Storyblok into your project using Svelte.",
+  "author": "Storyblok",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/storyblok/storyblok-svelte.git"
+  },
+  "bugs": {
+    "url": "https://github.com/storyblok/storyblok-svelte/issues"
+  },
+  "sideEffects": [
+    "**/*.css"
+  ],
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "svelte": "./dist/index.js",
+      "import": "./dist/index.js",
+      "require": "./dist/index.js"
+    }
+  },
+  "main": "./dist/index.js",
```

Comment:
@edodusi @roberto-butti is the build filename configurable? Should be `./dist/storyblok-svelte.js` to keep it consistent with other SDKs

Reply:
I tried with some custom configs and got many errors, went through some discussions for reference: https://github.com/sveltejs/kit/discussions/11429

Do you think this is important? If it is I'm gonna need some time to investigate this.

Reply:
If it's configurable, would be great for consistency. But no blocker if it's going to require more time atm